doc: describe more about troubleshooting tips

dongsupark opened the pull request:
Describe more about troubleshooting tips, mainly about setting up nodes as well as requirements. Fix typos and change names too.

Closes https://github.com/kinvolk/kube-spawn/issues/81

robertgzr commented on README.md:
````diff
@@ -49,6 +49,8 @@ $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn setup --image=coreos --n
 ```
 
+Now that you're up and running, this is how you can start using it.
+
````
I think this still needs something here ^^

dongsupark replied:
@robertgzr I just changed it simply to `"Now that you're up and running, you can start using it."`

robertgzr commented on doc/troubleshooting.md:
```diff
@@ -30,7 +60,7 @@ in another version is totally fine.
 don't encourage it.
 
-You can pass `kubeadm-nspawn` an alternative `systemd-nspawn` binary by setting the
+You can pass `kube-spawn` an alternative `systemd-nspawn` binary by setting the
```
nice job catching these :)

dongsupark commented:
For better readability, I moved the section of `Deploying to your local cluster` to the next of  `Quickstart`, and added a line `"kubectl get pods"` to the example. Now readers would get better impression of what to do after setting up nodes.
Added `(optional)` to the title of `Running local Kubernetes builds`, as it's not always necessary.
Added a line `"sudo iptables -P FORWARD ACCEPT"` to the example of `SELinux and firewalld`.

blixtra commented on README.md:
```diff
@@ -49,7 +49,23 @@ $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn setup --image=coreos --n
+
+## Deploying to your local cluster
+
+`kube-spawn` creates `tmp/` inside the directory you run it in.
```
Is we are now creating the `.kube-spawn` directory, why don't we just put that there too?

dongsupark replied:
@blixtra This will be done via https://github.com/kinvolk/kube-spawn/pull/102

blixtra replied:
I guess this needs to be updated now.

dongsupark replied:
@blixtra Done. Rebased and updated the document again.
